packages/marketplace/src/components/ui/analytics/billing/cost-explorer/cost-filter-form.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { Formik, Form, DatePicker } from '@reapit/elements'
+import FormikAutoSave from '@/components/hocs/formik-auto-save'
+import { CostFilterFormValues } from './cost-explorer'
+
+export type CostFilterFormProps = {
+  initialValues: CostFilterFormValues
+  onSave: (values: CostFilterFormValues) => void
+}
+
+export const handleAutoSave = (onSave: (values: CostFilterFormValues) => void) => {
+  return (values: CostFilterFormValues) => {
+    onSave(values)
+  }
+}
+
+const CostFilterForm: React.FC<CostFilterFormProps> = ({ initialValues, onSave }) => {
+  return (
+    <Formik initialValues={initialValues} onSubmit={() => {}}>
+      <Form>
+        <DatePicker
+          id="createdMonth"
+          name="createdMonth"
+          useCustomInput={false}
+          reactDatePickerProps={{
+            showMonthYearPicker: true,
+            dateFormat: 'MMMM YYYY',
```

Comment:
should you use const which you has been defined

Reply:
I mean this one 
![image](https://user-images.githubusercontent.com/22660833/80590626-709e7c80-8a46-11ea-94d7-63f2891e2e95.png)